airbyte-integrations/connectors/source-chargify/source_chargify/schemas/invoices.json

```
@@ -0,0 +1,501 @@
+                        "type" : ["integer", "null"]
+                    },
+                    "custom_item" : {
+                        "type" : "boolean"
+                    }
+                }
+            }
+        },
+        "discounts" : {
+            "type" : "array",
+            "items": {
+                "type" : "object",
+                "properties": {
+                    "uid" : {
+                        "type" : "string"
+                    },
+                    "title" : {
+                        "type" : "string"
+                    },
+                    "code" : {
+                        "type" : "string"
+                    },
+                    "source_type" : {
+                        "type" : "string"
+                    },
+                    "source_id" : {
+                        "type" : "integer"
+                    },
+                    "discount_id" : {
+                        "type" : "string"
+                    },
+                    "percentage" : {
+                        "type" : "string"
+                    },
+                    "eligible_amount" : {
+                        "type" : "string"
+                    },
+                    "discount_amount" : {
+                        "type" : "string"
+                    },
+                    "line_item_breakouts" : {
+                        "type" : "array",
+                        "items" : {
+                            "type" : "object",
+                            "properties": {
+                                "uid" : {
+                                    "type" : "string"
+                                },
+                                "eligible_amount" : {
+                                    "type" : "string"
+                                },
+                                "discount_amount" : {
+                                    "type" : "string"
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "taxes" : {
+            "type" : "array",
+            "items": {
+                "type" : "object",
+                "properties" : {
+                    "uid" : {
+                        "type" : "string"
+                    },
+                    "title" : {
+                        "type" : "string"
+                    },
+                    "source_type" : {
+                        "type" : "string"
+                    },
+                    "source_id" : {
+                        "type" : "integer"
+                    },
+                    "percentage" : {
+                        "type" : "string"
+                    },
+                    "taxable_amount" : {
+                        "type" : "string"
+                    },
+                    "discount_amount" : {
+                        "type" : "string"
+                    }
```

We're missing the `line_item_breakouts` array of objects mentioned in the [endpoint doc](https://developers.chargify.com/docs/api-docs/aa24a44561ce2-list-invoices#response-body):
```suggestion
                    },
                    "line_item_breakouts" : {
                      "type" : "array",
                      "items" : {
                        "type" : "object",
                        "additionalProperties" : true,
                        "properties" : {
                          "uid" : {
                          "type" : "string"
                          },
                          "taxable_amount" : {
                          "type" : "string"
                          },
                          "tax_amount" : {
                          "type" : "string"
                          }
                        }
                      }
```